### PR TITLE
Handles reson not listed

### PR DESCRIPTION
**Pull request by jbarnsley10**
Removed one of breach reasons

### CODE REVIEW

File: app/lib/email-helper.js [hunk 1]
```diff
@@ -11,7 +11,11 @@ const buildInBreach = data => {
     details = `Breach reported for Dog ${data?.dogChosen?.indexNumber}\n`
   }
   details += 'Reasons:\n'
-  details += data?.dogBreaches?.map(breach => ` - ${breach}\n`).join('')
+  if (!data?.dogBreaches) {
+    details += `${data?.details}\n`
+  } else {
+    details += data?.dogBreaches?.map(breach => ` - ${breach}\n`).join('')
+  }
```
Comment (whitewaterdesign) [inline app/lib/email-helper.js:18]: pretty sure you'll need some more coverage with all the optional chaining

Comment (jbarnsley10) [reply app/lib/email-helper.js:18]: I added an extra test to cover it

**Review by whitewaterdesign — APPROVED** [review]
Looks good to me

**Review by miriyalanarahari — APPROVED** [review]
Looks good.

**Pull request merged**